### DISCUSSION TITLE
Whatsapp records

### DISCUSSION THREAD

**Discussion by thiagocaparelli** [sepinf-inc/IPED#396] [Q&A] [answered]
Hello everyone.
Is there a way to processing whatsapp records on the tool? 
For example, I have two records from different numbers and I would like to compare groups, geolocalization, locations through IPs, etc. 
Beside that generating grafos, links between numbers and so on.
Anyway, We processed but is only indexing medias. 
Would you suggest a better aproach using IPED if they have one.

Thanks.

**Comment by lfcnassif** [comment, marked as answer]
Not sure what you mean by records, to me records are specific messages (text, media, geo, contact messages, so on...).

If you process a whatsapp database, shared locations sent in conversations are shown on map tab. Contacts are also extracted. I think IP addresses are not saved in WA database. Individual messages/records are extracted and could be viewed in "Instant Messages" category, you can sort them on table to have a "timeline" view of messages sent to different recipients. If you click on a group chat, in metadata tab, in the third vertical subtab, you can see group members, title and other group info.

And we have the communication graph you already know.

**Reply by lfcnassif** [reply]
You must process wa.db and msgstore.db files from Android or iPhone ContactsV2.sqlite and ChatStorage.sqlite files to process WhatsApp info correctly.

**Reply by thiagocaparelli** [reply]
First of all, thanks for the answer.
When I say "records" I mean this https://www.whatsapp.com/records/login/?locale=pt_BR.

**Reply by lfcnassif** [reply]
No, The tool does not process account or login records. Iped process whatsapp contact and message databases extracted from cellphones or HDDs.
